build.csx

```diff
@@ -148,32 +150,33 @@ targets.Add(
     DependsOn("build", "testsDirectory"),
     () =>
     {
-        RunTests(unitTests);
-        RunDotNetTests(netstdUnitTestDirectories);
+        RunDotNetTests(unitTestDirectories);
+        RunTests(analyzerTestAssemblies);
     });
 
 targets.Add(
+    "analyzer",
+    DependsOn("build", "testsDirectory"),
+    () => RunTests(analyzerTestAssemblies));

```

Comment:
Oops. In fact, I intended to remove it from the `unit` target. I thought it would make sense to be able to test only the analyzer or only the library.